src/test/resources/config/application.yml

```diff
@@ -46,6 +47,8 @@ spring:
             hibernate.cache.use_query_cache: false
             hibernate.generate_statistics: true
             hibernate.hbm2ddl.auto: validate
+            # see org.hibernate.cfg.AvailableSettings.PREFER_GENERATOR_NAME_AS_DEFAULT_SEQUENCE_NAME
+            hibernate.model.generator_name_as_sequence_name: false
```

Comment:
and what about prod profile? I guess the same changes must be done in application-prod.yml

Please try to save some entity using rest api and you will see result

Reply:
Indeed. Fixed.

Reply:
Done